.env.example

```
@@ -1,3 +1,12 @@
-VITE_ENV_SALT="generatayoursalt"
\ No newline at end of file
+VITE_ENV_SALT="generatayoursalt"
+
+# Auth0 SPA (public values only — never put client secrets in frontend env)
+VITE_AUTH0_DOMAIN="your-tenant.auth0.com"
+VITE_AUTH0_CLIENT_ID="your-spa-client-id"
+VITE_AUTH0_AUDIENCE="webuddhist-backend"
+VITE_AUTH0_SMS_CONNECTION="sms"
+VITE_AUTH0_GOOGLE_CONNECTION="google-oauth2"
+
+VITE_LOCATIONIQ_TOKEN="your-locationiq-token"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)